test/PluginManagerSuite.ts

```diff
@@ -1114,21 +1114,15 @@ describe("PluginManager:", function() {
 		});
 
 		it("cannot install multiple package concurrently", async function() {
-			// I expect this to take some time...
-			const installation1 = manager.installFromNpm("moment");
-
-			// so I expect a concurrent installation to fail...
 			const pluginSourcePath = path.join(__dirname, "my-basic-plugin");
+			const installation1 = manager.installFromNpm("moment");
 			const installation2 = manager.installFromPath(pluginSourcePath);
-
 			try {
-				await installation2;
-			} catch (err) {
-				await installation1;
-				return;
+				await Promise.all([ installation1, installation2 ]);
```

Comment:
I see that with this change the next test fails. Maybe because one of these promise is not yet completed?
A possible solution can be to use `Promise.allSettled` instead of `Promise.all`?
So that we are sure that all installation are completed and there aren't any pending installation.
What do you think?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled